src/articulation.ts

```diff
@@ -298,18 +292,20 @@ export class Articulation extends Modifier {
    * @param type entry in `Vex.Flow.articulationCodes` in `tables.ts` or Glyph code.
    *
    * Notes (by default):
-   * - Glyph codes ending with 'Above' will be positioned ABOVE
-   * - Glyph codes ending with 'Below' will be positioned BELOW
+   * - Even codes will be positioned ABOVE
+   * - Odd codes will be positioned BELOW
```

Comment:
Maybe we need to reference the appropriate SMuFL glyph page here? It seems arbitrary that Even is above and Odd is below, but if you look at the SMuFL code points, it makes more sense.